.github/workflows/create-jira-fix-issue.yml

```
@@ -1,34 +1,32 @@
 on:
   issues:
     types:
       - opened
 
-env:
-  ISSUE_HEADER : "[BE]"
-
 jobs:
   create-issue:
-    name: Create Jira issue
+    if : "startsWith(github.event.issue.title, 'Fix')"
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)